app/google-services.json

```diff
@@ -2,7 +2,8 @@
   "project_info": {
     "project_number": "134067572010",
     "project_id": "watpool",
-    "storage_bucket": "watpool.appspot.com"
+    "storage_bucket": "watpool.appspot.com",
+    "firebase_url": "https://watpool-default-rtdb.firebaseio.com/"
```

Comment:
Forgot to mention...this isn't needed anymore since we're using firestore. But we can remove this in a future PR